Compentio.SourceMapper.Generator/Generators/CodeSourceGenerator.cs

```diff
@@ -54,11 +58,23 @@ internal void GenerateDependencyInjectionExtensions(GeneratorExecutionContext co
             context.AddSource($"{_sourcesMetadata.DependencyInjection.DependencyInjectionClassName}.cs", SourceText.From(result.GeneratedCode, Encoding.UTF8));
         }
 
+        private void AddNoDependencyInjectionDiagnostic(GeneratorExecutionContext context)
+        {
+            _diagnostics.Add(new DiagnosticsInfo()
+            {
+                DiagnosticDescriptor = SourceMapperDescriptors.DependencyInjectionNotUsed,
+                Exception = new ExtException(AppStrings.DependencyInjectionNotUsedException, context.Compilation.AssemblyName)
```

Comment:
`DependencyInjectionNotUsedException` instead of `ExtException`?

Reply:
I changed name of the extended exception class to DependencyInjectionException.